lighthouse-core/lib/i18n/locales/ar.json

```diff
@@ -1310,17 +1340,29 @@
   "lighthouse-core/config/default-config.js | a11yTablesListsVideoGroupTitle": {
     "message": "الجداول والقوائم"
   },
+  "lighthouse-core/config/default-config.js | bestPracticesBrowserCompatGroupTitle": {
+    "message": "توافق المتصفّح"
+  },
   "lighthouse-core/config/default-config.js | bestPracticesCategoryTitle": {
     "message": "أفضل الممارسات"
   },
+  "lighthouse-core/config/default-config.js | bestPracticesGeneralGroupTitle": {
+    "message": "عام"
+  },
+  "lighthouse-core/config/default-config.js | bestPracticesTrustSafetyGroupTitle": {
+    "message": "الثقة والأمان"
+  },
+  "lighthouse-core/config/default-config.js | bestPracticesUXGroupTitle": {
+    "message": "انطباع المستخدم"
+  },
   "lighthouse-core/config/default-config.js | budgetsGroupDescription": {
     "message": "تضبط ميزانيات الأداء معايير لأداء موقعك الإلكتروني."
   },
   "lighthouse-core/config/default-config.js | budgetsGroupTitle": {
     "message": "الميزانيات"
   },
   "lighthouse-core/config/default-config.js | diagnosticsGroupDescription": {
-    "message": "مزيد من المعلومات حول أداء تطبيقك لا تؤثر هذه الأرقام [ بشكل مباشر في ](https://github.com/GoogleChrome/lighthouse/blob/d2ec9ffbb21de9ad1a0f86ed24575eda32c796f0/docs/scoring.md#how-are-the-scores-weighted) نتيجة الأداء."
+    "message": "مزيد من المعلومات حول أداء تطبيقك لا تؤثر هذه الأرقام [ بشكل مباشر في ](https://web.dev/performance-scoring/) نتيجة الأداء."
```

Comment:
wait, this is totally not gonna work as markdown link 😆

Reply:
> wait, this is totally not gonna work as markdown link 😆

oh no, I wonder if something's wrong with our ctc -> lhl conversion...

Reply:
That's just how github is with rtl. It works out in the end.
![image](https://user-images.githubusercontent.com/6392995/82008705-45a95080-9622-11ea-816b-892face090d5.png)

Reply:
> That's just how github is with rtl. It works out in the end.

ok, whew! Thanks @exterkamp

Reply:
but it's like randomly in the middle of the string...OK github weird stuff haha

well phew!